### PR TITLE
MCS-859 Fixed a bug with retrieving position and rotation from the output metadata

### PR DESCRIPTION
**Pull request by ThomasSchellenbergNextCentury**
Remembering to run the integration tests is important

### CODE REVIEW

**Review by brianpippin — APPROVED** [review]

**Review by deanwetherby — APPROVED** [review]

File: mcs_automated_ingest.py [hunk 1]
```diff
@@ -42,10 +43,11 @@ def process_message(message, message_type, db_string):
                 mcs_scene_ingest.automated_scene_ingest_file(
                     basename, "", db_string)
         except Exception as e:
+            logging.info(f"Error {traceback.format_exc()}")
```
Comment (kdrumm) [inline mcs_automated_ingest.py:46]: I'm fairly sure the logging system has built in support for logging execptions which we should use unless you are trying to do something very specific.  I can try to find an example.

Comment (kdrumm) [reply mcs_automated_ingest.py]: `logging.info("Exception: ", exc_info=e)` or `logging.info('blah', exc_info=True)`

I've never used the later, but probably should have.

Comment (deanwetherby) [reply mcs_automated_ingest.py:46]: `logging.error()`?

Comment (brianpippin) [reply mcs_automated_ingest.py:46]: I am just trying to get stacktrace into the message being sent to the sqs queue, this was an example i had found.  I also saw this:  sys.exc_info()[2] but that seemed not as clean to me.

Comment (brianpippin) [reply mcs_automated_ingest.py]: I did change it to logging.error

Comment (deanwetherby) [reply mcs_automated_ingest.py]: or logging.exception ... not sure exactly what you're after

**Review by kdrumm — COMMENTED** [review]
.. I don't know how to work github comments sometimes...

File: mcs_automated_ingest.py (hunk 1 above)
Comment (kdrumm) [reply mcs_automated_ingest.py]: using the exc_info paramater will add the stack trace and format it based on hwo the logger wants to.

**Comment by brianpippin** [comment]
> .. I don't know how to work github comments sometimes...

Will that work though for the changes on line 50 as well?

**Comment by kdrumm** [comment]
> > .. I don't know how to work github comments sometimes...
> 
> Will that work though for the changes on line 50 as well?

Is the goal to ensure the formatting is absolutely identical?  If so, then no, it won't do that and we should keep what we have.

**Comment by brianpippin** [comment]
> > > .. I don't know how to work github comments sometimes...
> > 
> > 
> > Will that work though for the changes on line 50 as well?
> 
> Is the goal to ensure the formatting is absolutely identical? If so, then no, it won't do that and we should keep what we have.

The goal is to get the stacktrace into the error queue so we have it immediately, if there is another way to do that I am fine for changing it, I even think we can remote the logging.error potentially, but the place it will be looked at the most is in the SQS error queue.

**Review by kdrumm — APPROVED** [review]
Either way with the logging issue, its not worth holding this up on.  I do think we should use the logger features, but more on a theoretical level than any practical level.  If everyone disagrees, we should move forward.

**Comment by brianpippin** [comment]
> Either way with the logging issue, its not worth holding this up on. I do think we should use the logger features, but more on a theoretical level than any practical level. If everyone disagrees, we should move forward.

I removed the one logging statement and left in the error going to the SQS queue so we can revisit if/how we want to log the error.

**Pull request merged**